batchflow/once_pipeline.py

```diff
@@ -114,6 +114,35 @@ def init_variable(self, name, default=None, lock=True, **kwargs):
         self.pipeline.variables.create(name, default, lock=lock, pipeline=self, **kwargs)
         return self
 
+    def init_variables(self, *variables):
+        """ Create several variables
+
+        Parameters
+        ----------
+        variables : dict or tuple
+            if dict
+                key : str - a variable name,
+                value : dict -  a variable value and init params (see :meth:`.init_variable`)
+            if tuple, contains variable names which will have None as default values
+
+        Returns
+        -------
+        self - in order to use it in the pipeline chains
+
+        Examples
+        --------
+        >>> pp = dataset.p
+                    .init_variables({"loss_history": dict(default=[]),
+                                     "predictions", dict(default=[])})
+                    .init_variables("metrics", "counter", "worst_prediction")
+                    .load('/some/path', fmt='blosc')
+                    .train_resnet()
+        """
+        if len(variables) == 1:
```

Comment:
did you want to check `isinstance(variables, dict)`?

Reply:
Nope, `variables` is always a tuple here